tests/integration/s2n_handshake_test_s_server.py

```diff
@@ -318,22 +321,33 @@ def elliptic_curve_test(host, port):
                     failed = 1
     return failed
 
+def str2bool(v):
+    if v.lower() in ('yes', 'true', 't', 'y', '1'):
+        return True
+    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
 
 def main():
     parser = argparse.ArgumentParser(description='Runs TLS server integration tests against Openssl s_server using s2nc')
     parser.add_argument('host', help='The host for s2nc to connect to')
     parser.add_argument('port', type=int, help='The port for s_server to bind to')
+    parser.add_argument('use_corked_io', type=str2bool, help='Turn corked IO on/off')
```

Comment:
I think makes more sense as an optional argument that takes no input: `--use-corked-io` rather than positonal.

`parser.add_argument('--use-corked-io', action='store_true', help='Turn corked IO on/off')`

Reply:
Done